webcompat/api/endpoints.py

```diff
@@ -128,7 +130,8 @@ def get_issue_category(issue_category):
     # For paginated results on the /issues page,
     # see /issues/search/needstriage.
     elif issue_category == 'new':
-        abort(301)
+        redirect(url_for('.get_issue_category',
+                         issue_category='needstriage'), 301)
```

Comment:
I think you probably want `redirect(url_for('api.get_issue_category',` because of the `api` Blueprint... but I haven't tested.